base/experimental.jl

```diff
@@ -494,4 +494,13 @@ function entrypoint(@nospecialize(argt::Type))
     nothing
 end
 
+"""
+    Base.Experimental.disable_new_worlds()
+
+Mark that no new worlds (methods additions, deletions, etc) are permitted to be
```

Comment:
It's probably worth mentioning that this transition is permanent (if I understand correctly)